clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -114,13 +114,37 @@ class ServerGroupController {
   @RequestMapping(method = RequestMethod.GET)
   List list(@PathVariable String application,
             @RequestParam(required = false, value = 'expand', defaultValue = 'false') String expand,
-            @RequestParam(required = false, value = 'cloudProvider') String cloudProvider) {
+            @RequestParam(required = false, value = 'cloudProvider') String cloudProvider,
+            @RequestParam(required = false, value = 'clusters') Collection<String> clusters) {
+    if (clusters) {
+      if (clusters.empty) {
```

Comment:
this would never happen (thx groovytruth) since `if (clusters)` above would be false